.ci/docker/test/verify.sh

```diff
@@ -90,6 +90,16 @@ else
     exit 1
 fi
 
+BASE_IMAGE_REGEX='ARG BASE_IMAGE="([^"]+)"'
+DOCKERFILE=$(<"${BASH_SOURCE%/*}/../../../${DOCKERFILENAME}")
```

Comment:
:upside_down_face: We could just define this once (see line 84). Would then also be a good opportunity to gracefully handle the case that this file doesn't exist.

Reply:
right, came with copy pasting 😄